e2e/support/helpers/e2e-qa-databases-helpers.js

```diff
@@ -103,12 +103,12 @@ function assertOnDatabaseMetadata(engine) {
 }
 
 function recursiveCheck(id, i = 0) {
-  // Let's not wait more than 5s for the sync to finish
-  if (i === 20) {
+  // Let's not wait more than 20s for the sync to finish
+  if (i === 40) {
     throw new Error("The sync is taking too long. Something is wrong.");
   }
 
-  cy.wait(250);
+  cy.wait(500);
```

Comment:
Do you know what the reason was for needing to increase the wait time?

Reply:
I've been getting occasional failures, even locally, not just in this branch, sometimes I have to try 2-3 times to get it to sync fast enough. 🤷

Reply:
Could just be that there is more stuff in the snapshot now? Would that make sense?